docs/nachet-inference-documentation.md

```diff
@@ -1,12 +1,24 @@
 # Inference Request with n pipelines
 
+## Executive summary
+Following the objective of milestones 1, Nachet back end need to deploy a
+functionality to switch in between different model. However, a model can call
+another model for more accurate result. Therefore creating a pipeline of models.
```

Comment:
"However, a model can call another model for more accurate result." - none of this is true

pipeline is introduced but not really explained (assumptions)